sphinx_immaterial/custom_admonition.py

```diff
@@ -0,0 +1,46 @@
+"""This module inherits from the generic ``admonition`` directive and makes the
+title optional."""
+from docutils import nodes
+from docutils.parsers.rst.roles import set_classes
+from docutils.parsers.rst.directives import admonitions
+from sphinx.application import Sphinx
+
+__version__ = "0.0.1"
+
+
+class noTitleAdmonition(admonitions.BaseAdmonition):
```

Comment:
Can you rename this to `NoTitleAdmonition`?

Reply:
Good catch! I'm surprised the linters didn't pick up on that.

```suggestion
class NoTitleAdmonition(admonitions.BaseAdmonition):
```